<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsSimulatorBasicsComponent</class>
 <widget class="QFrame" name="CSettingsSimulatorBasicsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QPushButton" name="pb_CopyDefaults">
     <property name="text">
      <string>copy defaults</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pb_ModelFileDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="le_ModelDirectory">
     <property name="placeholderText">
      <string>Model directory path</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignTop">
    <widget class="QLabel" name="lbl_ExcludeDirectories">
     <property name="toolTip">
      <string>Excluded from model loading</string>
     </property>
     <property name="text">
      <string>Exclude directories:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pb_Save">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pb_Reset">
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_ModelDirectory">
     <property name="text">
      <string>Model directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="pb_ExcludeFileDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_SimulatorDirectory">
     <property name="text">
      <string>Simulator directory;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="le_SimulatorDirectory">
     <property name="placeholderText">
      <string>Simulator directory path</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="4">
    <widget class="QPlainTextEdit" name="pte_ExcludeDirectories">
     <property name="documentTitle">
      <string>Excluded directory patterns</string>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="placeholderText">
      <string>Excluded directory patterns</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pb_SimulatorFileDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4" alignment="Qt::AlignRight">
    <widget class="BlackGui::Components::CSimulatorSelector" name="comp_SimulatorSelector">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
